snap/snapenv/snapenv.go

```diff
@@ -63,6 +63,7 @@ func GetBasicSnapEnvVars(desc interface{}) []string {
 	return fillSnapEnvVars(desc, []string{
 		"SNAP={{.SnapPath}}",
 		"SNAP_DATA=/var{{.SnapPath}}",
+		"SNAP_SHARED_DATA=/var{{.SnapBasePath}}/shared",
```

Comment:
That would involve adding `stripGlobalRootDir` as a template function which makes this a little less pretty. Is that okay?